threeplay.go

```diff
@@ -25,17 +25,11 @@ const (
 	HTML OutputFormat = "html"
 )
 
-// HTTPClient interface for http clients used in the 3Play client
-type HTTPClient interface {
-	Get(string) (*http.Response, error)
-	PostForm(string, url.Values) (*http.Response, error)
-}
-
 // Client 3Play Media API client
 type Client struct {
-	apiKey    string
-	apiSecret string
-	client    HTTPClient
+	apiKey     string
+	apiSecret  string
+	HTTPClient *http.Client
```

Comment:
I mean, place the tests in the same package (i.e. use `package threeplay` in the test file, instead of `package threeplay_test`).